decko-rails/config/routes.rb

```diff
@@ -1,11 +1,7 @@
 # -*- encoding : utf-8 -*-
 
-Rails.application.routes.draw do
+Decko::Engine.routes.draw do
```

Comment:
This one looks suspicious.  It may have been in card-gem, but you wanted to put it off to this branch maybe?